src/core/ModuleManager.sol

```diff
@@ -34,6 +60,12 @@ abstract contract ModuleManager is IRegistry, ResolverManager {
 
     mapping(address moduleAddress => ModuleRecord moduleRecord) internal $moduleAddrToRecords;
 
+    FactoryTrampolin private immutable FACTORY_TRAMPOLIN;
```

Comment:
should I make this public? could be helpful if specialized factories want to do access control for the trampolin. would be better devex?!

Reply:
yeah that makes sense imo